docs/t-sql/language-elements/bitwise-not-transact-sql.md

```diff
@@ -63,11 +63,9 @@ monikerRange: ">=aps-pdw-2016||=azuresqldb-current||=azure-sqldw-latest||>=sql-s
  The following example creates a table using the **int** data type to store the values and inserts the two values into one row.  
   
 ```  
-CREATE TABLE bitwise  
-(   
-a_int_value int NOT NULL,  
-b_int_value int NOT NULL  
-);  
+CREATE TABLE #bitwise (  

```

Comment:
Hello. You changed the table to #bitwise but then the insert is into bitwise which will fail. Can you remove the #4880 

```suggestion
CREATE TABLE bitwise (  
```

Reply:
Sure. I will update it

Reply:
It's updated now